src/test/lambda/wizards/samDeployWizard.test.ts

```diff
@@ -187,6 +208,219 @@ describe('SamDeployWizard', async () => {
         })
     })
 
+    describe('PARAMETER_OVERRIDES', async () => {
+        function makeFakeRegionProvider(): RegionProvider {
```

Comment:
Why not use `MockRegionProvider`?

Reply:
Good catch, I forgot to replace this when I added `MockRegionProvider`. Will do.